tests/syft/grid/duet_test.py

```diff
@@ -1,78 +1,77 @@
 # third party
-import pytest
```

Comment:
Q: Nothing from here works?

Reply:
This has been removed, Duet has been removed we only have WebRTCDuet now